java-checks/src/main/java/org/sonar/java/checks/EscapedUnicodeCharactersCheck.java

```diff
@@ -54,6 +54,9 @@ public void visitNode(Tree node) {
       return;
     }
     String value = LiteralUtils.trimQuotes(((LiteralTree) node).value());
+    if (node.is(Kind.TEXT_BLOCK)) {
+      value = value.replaceAll("(\r?\n|\r)\\s*+", "");
```

Comment:
Is the `+`  operator necessary here? The '*' just before already covers the same case. Or is the `+` supposed o apply to the entire expression?

Reply:
Yes, possessive quantifier is odd in this context, copy/paste issue